dotnet/src/Connectors/Connectors.OpenAI/AudioToText/OpenAIAudioToTextExecutionSettings.cs

```diff
@@ -146,6 +156,8 @@ public override PromptExecutionSettings Clone()
 
     #region private ================================================================================
 
+    private const string DefaultFilename = "file.mp3";
```

Comment:
This default file name is required because of an issue in the Azure AI SDK which Dmytro has raised here: https://github.com/Azure/azure-sdk-for-net/pull/40078#discussion_r1480392822

We should leave a note that we want to revisit this when the Azure SDK team have made their change.

Reply:
@markwallace-microsoft I think we can create an issue to keep track of this instead of leaving a note in the code. 👍🏻